aspnetcore/security/anti-request-forgery.md

```diff
@@ -4,142 +4,230 @@ author: steve-smith
+1. The request runs on the `www.good-banking-site.com` server with the user's authentication context and can perform any action that an authenticated user is allowed to perform.
 
-This example requires the user to click the form button. The malicious page could:
+When the user selects the button to submit the form, the malicious site could:
 
 * Run a script that automatically submits the form.
 * Sends a form submission as an AJAX request. 
 * Use a hidden form with CSS. 
 
-Using SSL doesn't prevent a CSRF attack, the malicious site can send an `https://` request. 
+Using HTTPS doesn't prevent a CSRF attack. The malicious site can send an `https://www.good-banking-site.com/` request just as easily as it can send an insecure request.
+
+Some attacks target endpoints that respond to GET requests, in which case an image tag can be used to perform the action. This form of attack is common on forum sites that permit images but block JavaScript. Apps that change state on GET requests, where variables or resources are altered, are vulnerable to malicious attacks. **Do not respond to GET requests with state changes.**
+
+CSRF attacks are possible against web apps that use cookies for authentication because:
+
+* Browsers store cookies issued by a web app.
+* Stored cookies include session cookies for authenticated users.
+* Browsers send all of the cookies associated with a domain to the web app every request regardless of how the request to app was generated within the browser.
 
-Some attacks target site endpoints that respond to `GET` requests, in which case an image tag can be used to perform the action (this form of attack is common on forum sites that permit images but block JavaScript). Applications that change state with `GET` requests are vulnerable from malicious attacks.
+However, CSRF attacks aren't limited to exploiting cookies. For example, Basic and Digest authentication are also vulnerable. After a user signs in with Basic or Digest authentication, the browser automatically sends the credentials until the session&dagger; ends.
 
-CSRF attacks are possible against web sites that use cookies for authentication, because browsers send all relevant cookies to the destination web site. However, CSRF attacks are not limited to exploiting cookies. For example, Basic and Digest authentication are also vulnerable. After a user logs in with Basic or Digest authentication, the browser automatically sends the credentials until the session ends.
+&dagger;In this context, *session* refers to the client-side session during which the user is authenticated. It's unrelated to server-side sessions or [ASP.NET Core Session Middleware](xref:fundamentals/app-state).
 
-Note: In this context, *session* refers to the client-side session during which the user is authenticated. It's unrelated to server-side sessions or [session middleware](xref:fundamentals/app-state).
+Users can guard against CSRF vulnerabilities by taking simple precautions:
```

Comment:
"simple"? Remove that word. Clearing browser cookies is not simple for the majority of users.